static/app/views/monitors/components/cronsLandingPanel.tsx

```diff
@@ -1,22 +1,117 @@
+import {useState} from 'react';
 import styled from '@emotion/styled';
 
 import onboardingImg from 'sentry-images/spot/onboarding-preview.svg';
 
-import {LinkButton} from 'sentry/components/button';
+import {Button, LinkButton} from 'sentry/components/button';
 import ButtonBar from 'sentry/components/buttonBar';
 import OnboardingPanel from 'sentry/components/onboardingPanel';
+import Panel from 'sentry/components/panels/panel';
+import PanelBody from 'sentry/components/panels/panelBody';
+import {TabList, TabPanels, Tabs} from 'sentry/components/tabs';
+import {PlatformKey} from 'sentry/data/platformCategories';
+import {IconChevron} from 'sentry/icons';
 import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
 
 import {NewMonitorButton} from './newMonitorButton';
-import {PlatformPickerPanel} from './platformPickerPanel';
+import {
+  CRON_SDK_PLATFORMS,
+  PlatformPickerPanel,
+  SupportedPlatform,
+} from './platformPickerPanel';
+import {QuickStartProps} from './quickStartEntries';
+
+interface PlatformGuide {
+  Guide: React.ComponentType<QuickStartProps>;
+  title: string;
+}
+
+const platformGuides: Record<SupportedPlatform, PlatformGuide[]> = {
+  'python-celery': [
+    {
+      Guide: () => null,
```

Comment:
will be replacing these with actual quickstart guides

Reply:
do we need a `Coming soon!` or are these not visible?

Reply:
oh they are feature flagged, so only internal folk can see them